docs/operator-manual/applicationset/Generators-Git.md

```diff
@@ -443,3 +443,9 @@ stringData:
 ```
 
 After saving, please restart the ApplicationSet pod for the changes to take effect.
+
```

Comment:
I did add a section regarding this in the [project](https://argo-cd.readthedocs.io/en/latest/user-guide/projects/) section (and for some reason it's not being displayed in the `stable` release 🤔), although it probably makes sense to have this in the Git generator section as well...

Reply:
Ah, look at that :)

Not sure where it should actually belong, i kinda find it weird to have under "project", since the applicationset can't belong to a project anyway.
For me it belongs under Generators-git, since that's where it will be used. But perhaps a mention directly on Applicationset is good too.

Reply:
I'm happy either way, I just wanted to write it down somewhere so people knew how it worked.
Up to you 👍

Reply:
This makes sense to me 😄